.gitignore

```
@@ -102,7 +102,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
+stracker/.env
```

bunda değişiklik yapmaya gerek yok. olduğu gibi kalmalı ki başka .env dosyalarını da exclude edebilelim.
```suggestion
.env
```

---

gitignore dosyasına hiç dokunmamıştım, heralde kendisi değiştirdi. Commitlerken dikkat etmemişim